### DISCUSSION TITLE
Implementing Custom Claims

### DISCUSSION THREAD

**Discussion by ryan-hong** [gladly-team/next-firebase-auth#209] [Q&A]
First want to say this library is great, have used it to add SSR authentication to my NextJS project.  

I am curious how to implement custom claims with this specific library?  I have seen tutorials on creating the actual claims, but those are for projects that have a separate backend and frontend.  Since NextJS is "all-in-one", I am looking to see if anyone would be able to guide me on creating custom claims.

For example, I want to create a Role-Based Authentication Access so want to create custom claims for specific roles like "Admin" and "User", etc...

If anyone has done this before with NextJS, any help is welcome!

Thanks in advance.

**Comment by kmjennison** [comment]
In case you hadn't seen, you can access custom claims in the [AuthUser `claims` field](https://github.com/gladly-team/next-firebase-auth#authuser). Actually setting the claims is up to you (it's not something this package handles).

**Reply by ryan-hong** [reply]
yup i have sort of figured it out.  Im gonna hijack this comment with another question.  is `getFirebaseAdmin` the same as the dependency `firebase-admin`? i.e. has same features and can be accessed the same way? 

I am currently trying to use `getFirebaseAdmin().auth().getUserByEmail()` but doesn't seem to be working properly

**Reply by ryan-hong** [reply]
nvm realized i was doing it on client side
